demisto_sdk/commands/common/hook_validations/incident_field.py

```diff
@@ -354,9 +354,26 @@ def is_valid_required(self):
         # as can be seen in this pr: https://github.com/demisto/content/pull/5682
         required = self.current_file.get('required', False)
         if required:
-            error_msg = f'{self.file_path}: new incident fields can not be required' \
-                        f' due to a current platform limitation.'
+            error_msg = f'{self.file_path}: new incident fields can not be required.' \
```

Comment:
new fields can be required if they are associated with new incident type.

Reply:
I wish, but it is not true due to a platform limitation.
https://github.com/demisto/etc/issues/22069